### DISCUSSION TITLE
Export layer to PDF with layer

### DISCUSSION THREAD

**Discussion by depperdc** [jgraph/drawio#2458] [Ideas]
Hello.
I know that. AutoCAD can export layers to a **layered PDF**. Your diagrams.net is great. Did you think you could also export layers in PDF? Now everything is in one PDF layer. Thank you and I wish you much success.
Martin Beran

**Comment by JanB1** [comment]
I second this! I currently would have to use Visio or an other, alternative software to have this feature.
See also Issue/Feature Request jgraph/drawio#1561.
